aries_cloudagent/connections/base_manager.py

```diff
@@ -73,17 +69,19 @@ async def create_did_document(
             The prepared `DIDDoc` instance
 
         """
-
-        did_doc = DIDDoc(did=did_info.did)
+        if len(did_info.did.split(":")) < 2:
```

Comment:
With this line I evaluate if the DID is complete or not.
If the DID is not complete... It is needed to prefix it.